### ISSUE TITLE
Enabling xla on tensorflow-rocm with Rocm 5.2.3?

### ISSUE THREAD

**Issue by ffleader1** [ROCm/ROCm#1796]
I have a gfx1030 card l, Rocm 5.2.3 and currently running into xla issues with tensorflow.
My GPU can do pytorch model fine, can do tensorflow fine, without xla. With xla however, I got this error: ````bitcode module not found at ./opencl.bc````

Am I missing something during the install process (because I installed with ````rocm```` usecase).

How to get my device running tensorflow training with xla?
Thank you

**Comment by aoolmay** [comment]
Can confirm, error is reproducible on 6800XT and 6950XT.
Fresh installation: ROCm 5.2.3 + tensorflow-rocm 2.9.2 + Linux 5.11.0-27 (as per installation instructions)
Last working setup, ROCm 5.1.3 + tensorflow 2.9.1

Loss of XLA capability results in ~7% performance loss on most tensorflow workloads.

**Comment by Mushoz** [comment]
Same issue here. Any idea on when we can expect a fix?

**Comment by macmv** [comment]
I've had the same issue (also running arch with a 6800xt), and setting `ROCM_PATH` to `/opt/rocm` fixed it for me.

**Comment by Epliz** [comment]
hit the same issue with a MI100 GPU, setting `ROCM_PATH` also helped

**Comment by Henry-Kr15** [comment]
Hey, sorry if this might be dumb, but how can I set the ROCM_PATH?

**Comment by nartmada** [comment]
Hi @ffleader1, please check latest ROCm Documentation and ROCm 6.0.0 to see if your query has been resolved.  If resolved, please close the ticket.  Thanks.

**Comment by ffleader1** [comment]
> Hi @ffleader1, please check latest ROCm Documentation and ROCm 6.0.0 to see if your query has been resolved. If resolved, please close the ticket. Thanks.

I am no longer have access to my original PC
But guessing the complaints were not that much anymore, I guess it's fixed.
